### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by Evert-Arends** [woocommerce/wc-api-python#56]
Hello,

As of a few months this module does not seem to work anymore, I found the culprit, as it seems to work when I include this header:
'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64; rv:80.0) Gecko/20100101 Firefox/80.0'. This explains why CURL works, but python requests not anymore.

Best regards,

Evert

**Comment by freddie-freeloader** [comment]
I second this. :disappointed: 

I get 403 status code for `print(wcapi.get("orders", params={"per_page": 20}).status_code)`.

**Comment by Evert-Arends** [comment]
@freddie-freeloader If you want a temporary fix, you can check out my branch, where I fixed it in a minimal way.
https://github.com/Evert-Arends/wc-api-python

If you want the full fix check out my pr: #57

**Comment by freddie-freeloader** [comment]
@Evert-Arends Thank you for pointing that out! I forked your repository already but used the master branch. Dank u zeer! :relaxed:

**Comment by jasalt** [comment]
I was getting 401 authentication error with this but queries with the Postman/curl were working. Ended up simply using the Requests library with it's basic auth.

```
import requests
response = requests.get(API_URL,
                                           auth=(KEY,SECRET),
                                           # verify=False  # if testing dev server with self signed ssl 
                                          )
print(response.status_code)
```

This is basically what this wrapper library tries to help you do anyways while complicating things at the same time.. Hope the fix is on PIP soon, thank's.

**Comment by Evert-Arends** [comment]
Fixed in merged pr #57
